quantities/dimensions/CompoundDimensions.cs

```diff
@@ -1,31 +1,8 @@
 ﻿namespace Quantities.Dimensions;
 
-public interface IArea<out TLength> : ISquare<TLength>
-    where TLength : ILength
-{ /* marker interface */ }
-public interface IVolume<out TLength> : ICubic<TLength>
-    where TLength : ILength
-{ /* marker interface */ }
-public interface IVelocity : IDimension { /* marker interface */ }
-public interface IVelocity<out TLength, out TTime> : IPer<TLength, TTime>, IVelocity
-    where TLength : ILength
-    where TTime : ITime
-{ /* marker interface */ }
+public interface IArea : ISquare<ILength>, IDimension { /* marker interface */ }
```

Comment:
This version of the interface feels cleaner!

Reply:
Yeah, it was a real mess. Mostly because I wasn't sure yet what it would be all about. Now, it's slowly becoming clear that every dimensional marker should be non generic ("scalar"), whereas the relations amongst themselves use generic compounds of scalar values.